devlab/tests/generate_load.py

```diff
@@ -162,14 +241,28 @@ def switch_user(self, user, password, tenant):
                                           user, password, tenant,
                                           self.auth_url)
 
-    def create_users(self):
+
+class Prerequisites(BasePrerequisites):
+
+    def __init__(self, config, cloud_prefix='SRC'):
+        super(Prerequisites, self).__init__(config, cloud_prefix)
+        # will be filled during create all networking step
+        self.ext_net_id = None
+        # object of Prerequisites for dst cloud
+        self.dst_cloud = None
+        self.clean_tools = CleanEnv(config, cloud_prefix)
+
+    @clean_if_exists
+    def create_users(self, users=None):
         def get_params_for_user_creating(_user):
```

Comment:
create_users should be used for user creating. I do not see cases, when this method will be re-used in other methods.